api/src/main/java/org/pmiops/workbench/leonardo/LeonardoApiClient.java

```diff
@@ -29,9 +29,6 @@ public interface LeonardoApiClient {
   /** lists all notebook runtimes as the appengine SA, to be used only for admin operations */
   List<LeonardoListRuntimeResponse> listRuntimesByProjectAsService(String googleProject);
 
-  List<LeonardoListRuntimeResponse> listRuntimesByProject(
```

Comment:
Was only used [as a fallback for getRuntime](https://github.com/all-of-us/workbench/pull/9209/files#diff-f58a0669dc3d1b961de4885cc579453289f725063ababacf7b1aca757f322573L121) but no longer relevant.